SamplesCommon/SamplesCommon/SamplesCommon.csproj

```diff
@@ -160,7 +160,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-      <Version>6.1.9</Version>
+      <Version>6.2.0-preview1-26926-04</Version>
```

Comment:
Why are we using a preview version of .NETCore UWP stuff? That looks unintentional.